### DISCUSSION TITLE
Cannot be generated because EJB Lite classes are not available on project classpath.

### DISCUSSION THREAD

**Discussion by akaouane** [apache/netbeans#7096] [General]
Hi every one.
When I try to use the Wizard "New JSF Pages from Entity Classes", I get the Error message :
"Cannot be generated because EJB Lite classes are not available on project classpath."
I don't know what I have to do to correct this issue.
I use Apache Netbeans 21, Jakarta 10, OpenJdk 11, GlassFish 7.
Thanks in advance.

**Comment by matthiasblaesing** [comment]
This sounds like a duplicate of #7066. In the issue I refer a PR that should fix the issue and also a nightly build. Please see if that helps.

**Comment by akaouane** [comment]
I think the problem apear on Linux and not on Windows.
I created a new virtualbox instance (windows 10) and I installed apache netbeans 21, and all things work fine.
But on my linux (Ubuntu 22.04), I get problems when trying to create JSF pages from entity classes:
1- I get classes in the list that not exists in my project
2- I get the message error : 
![Screenshot from 2024-02-22 22-59-40](https://github.com/apache/netbeans/assets/101929111/5576377e-e41a-4e21-8192-24defd9ffbeb)

**Reply by matthiasblaesing** [reply]
Sorry, but I tell you what I tell most people: Prepare a test case, that reproduces your problem. Else this stays speculation and can't be fixed.
